packages/repl/src/lib/workers/bundler/index.js

```diff
@@ -434,7 +447,7 @@ async function bundle({ uid, files }) {
 				format: 'iife',
 				name: 'SvelteComponent',
 				exports: 'named',
-				sourcemap: true
+				sourcemap: 'inline'
```

Comment:
will this show up in the JS Output tab? I don't know that I'd want it to. Can we leave it as `true` or should we set it to `false`?

Reply:
No. What's shown in the output tab is generated by the other worker, `compiler/index.js`.